test/functional/test.sh

```diff
@@ -59,6 +59,35 @@ do
     trap - EXIT
 done
 
+for siteInfo in $(cat test_template_sites);
+do
+    trap cleanup_initialized EXIT
+
+    flag=$(cut -d',' -f1 <<<"$siteInfo")
+    site=$(cut -d',' -f2 <<<"$siteInfo")
+
+    # print site name
+    echo "Running $site test"
+
+    # init and build based on flag
+    node ../../index.js init $site/tmp --template $flag
+    node ../../index.js build $site/tmp
+
+    # run our test script to compare html files
+    node testUtil/test.js compare $site --expectedPath expected --sitePath tmp/_site
+    
+    # remove generated files
+    rm -rf $site/tmp
+
+    if [ $? -ne 0 ]
```

Comment:
This is checking the return code of `rm`, but we should be checking the return code of `node testUtil/test.js`.